launch/psen_scan_v2.launch

```diff
@@ -33,23 +33,19 @@ along with this program.  If not, see <https://www.gnu.org/licenses/>.
   <arg name="frame_id" default="scanner" />
 
   <!-- Start angle of measurement in radian -->
-  <arg name="angle_start" default="0" />
+  <arg name="angle_start" default="$(eval -137.5 /180.0 * pi)" />
 
   <!-- End angle of measurement in radian -->
-  <arg name="angle_end" default="$(eval 275.0 /180.0 * pi)" />
-
-  <!-- Rotation of x-axis around the center in radian. -->
-  <arg name="x_axis_rotation" default="$(eval 137.5 / 180.0 * pi)" />
+  <arg name="angle_end" default="$(eval 137.5 /180.0 * pi)" />
 
   <node name="laser_scanner" type="psen_scan_v2_node" pkg="psen_scan_v2" output="screen" required="true">
-    <rosparam param="host_ip" subst_value="True">$(arg host_ip)</rosparam>
-    <rosparam param="host_udp_port_data" subst_value="True">$(arg host_udp_port_data)</rosparam>
-    <rosparam param="host_udp_port_control" subst_value="True">$(arg host_udp_port_control)</rosparam>
-    <rosparam param="sensor_ip" subst_value="True">$(arg sensor_ip)</rosparam>
-    <rosparam param="frame_id" subst_value="True">$(arg frame_id)</rosparam>
-    <rosparam param="angle_start" subst_value="True">$(arg angle_start)</rosparam>
-    <rosparam param="angle_end" subst_value="True">$(arg angle_end)</rosparam>
-    <rosparam param="x_axis_rotation" subst_value="True">$(arg x_axis_rotation)</rosparam>
+    <param name="host_ip" value="$(arg host_ip)" />
```

Comment:
What is this change doing?

Reply:
The `rosparam` syntax requires an additional `subst_value="True"` argument, to evaluate variables, the `param` syntax is shorter (and thus better imho). Do you see any advantages of the `rosparam` syntax? The only advantage is, that you can set multiple parameters / arrays as yaml inside `rosparam`, but we do not use that here...